sidekiq-instrumental.gemspec

```diff
@@ -23,7 +23,6 @@ Gem::Specification.new do |spec|
 
   spec.required_ruby_version = '>= 2.3'
 
-  spec.add_runtime_dependency 'instrumental_agent', '>= 0.13'
   spec.add_runtime_dependency 'sidekiq', '>= 4.0', '< 6.0'
```

Comment:
Are there known reasons why sidekiq should be less than 6? I'm going upgrade sidekiq on hs and account to 6.5.4 this week, since this version is introducing massive improvements in throuput.

Reply:
I'm not aware of any. If 6 version doesn't have any changes in middlewares api then we should be good.
Anyway we'll need to test it.

Reply:
Let's upgrade this dependency to be `< 7`.